t/06-blacklist.t

```diff
@@ -1,10 +1,13 @@
 #!/usr/bin/perl
 
+use LedgerSMB::Sysconfig;
```

Comment:
@ehuelsmann I'm not so concerned about the use of Sysconfig.pm here.
We ave other tests that also use it so the precedent is set